mmcv/ops/csrc/common/cuda/nms_cuda_kernel.cuh

```diff
@@ -72,4 +72,40 @@ __global__ void nms_cuda(const int n_boxes, const float iou_threshold,
     }
   }
 }
+
+__global__ void gather_keep_from_mask_parallize(
+    bool *keep, const unsigned long long *dev_mask, const int n_boxes) {
+  const int col_blocks = (n_boxes + threadsPerBlock - 1) / threadsPerBlock;
+  const int tid = threadIdx.x;
+
+  extern __shared__ unsigned long long remv[];
```

Comment:
remv is not a very clear shortname (I cannot get its meaning at the first glance). Better to rename it or add an explanation to tell readers whose abbreviation it is.